src/pages/categories.jsx

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categoriesSlice';
 
 export default function Categories() {
+  const dispatcher = useDispatch();
+
   return (
-    <button className="cat-button" type="button">
+    <button
+      className="cat-button"
+      type="button"
+      onClick={() => {
+        dispatcher(checkStatus());
```

Comment:
- [ ] Kindly ensure that when you dispatch checkStatus, a text "Under Construction is displayed on the page. Please see a video demo below:

https://www.loom.com/share/433d2288c1034eea9f60b6f0c7a4ee58